explorer/package.json

```diff
@@ -32,23 +32,25 @@
   },
   "main": "src/index.ts",
   "dependencies": {
+    "@types/express-pino-logger": "^4.0.2",
+    "@types/pino": "^5.15.5",
```

Comment:
Can this be moved to "devDependencies"?

Reply:
No, then we'd have no logging at all.

Reply:
This is just moving the type definition to dev dependencies. It's not used during runtime.